test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -91,14 +83,20 @@ public static void main(String[] args) throws Exception {
             Color botClr = robot.getPixelColor(point.x, point.y + (SQUARE_HEIGHT/2) - PIXEL_BUFFER);
 
             testImageCentering(leftClr, rightClr, topClr, botClr);
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException e) {
+            throw new RuntimeException("Unsupported LookAndFeel: " + e);
+        } catch (IOException e) {
+            //save image for troubleshooting
+            BufferedImage failImg = robot.createScreenCapture(new Rectangle(point.x - BUTTON_WIDTH / 2, point.y - BUTTON_HEIGHT / 2, BUTTON_WIDTH, BUTTON_HEIGHT));
+            ImageIO.write(failImg, "png", new File(testDir + "/fail_square.png"));
+            throw new RuntimeException("Failed image generation: " + e);
         } finally {
             // dispose frame when done testing for a LAF before continuing
             SwingUtilities.invokeAndWait(() -> frame.dispose());
```

Comment:
`frame` can be null here, if `generateImage` thrown the exception. You should check if frame is null in the lambda expression.